rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/RobotCommandHandle.py

```diff
@@ -537,6 +526,19 @@ def dist(self, A, B):
         assert(len(B) > 1)
         return math.sqrt((A[0] - B[0])**2 + (A[1] - B[1])**2)
 
+    def get_speed_limit(self, target_waypoint):
```

Comment:
Suggest minor refactor like this for clarity. Initializing a large max value and then updating it with a lower value in a list is the standard pattern for finding a minimum.

    def get_speed_limit(self, target_waypoint):
        approach_lane_limit = np.inf
        approach_lanes = target_waypoint.approach_lanes
        for lane_index in approach_lanes:
            lane = self.graph.get_lane(lane_index)
            lane_limit = lane.properties.speed_limit
            if lane_limit is not None:
                if lane_limit < approach_lane_limit:
                    appraoch_lane_limit = lane_limit
        return appraoch_lane_limit if appraoch_lane_limit != np.inf else 0.0

Reply:
Okay got it! Thanks for the suggestion, updated accordingly